### PR TITLE
Tailwindow -> Deleted deadlink

### PR DESCRIPTION
**Pull request by robvanbakel**
# Tailwindow

The link to Tailwindow was present, but pointed to a dead page.

Link: [Tailwindow ](https://component.tailwindow.com/)

#### Checklist:

- [x] I have performed a self-review of submitted resource and its follows the guidelines of the project.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -719,7 +719,6 @@
 | [Soft Components](https://soft-components-docs.web.app/)| A set of framework-agnostic web components based on neumorphic design. |
 | [Fast](https://www.fast.design/)| An interface system that can be used with modern Web Frameworks such as React, Vue and Angular. |
 | [LottieFiles ](https://lottiefiles.com/)| Interactive animations in many formats like json,gif and mp4, libraries and plugins for Web & Mobile . |
-| [Tailwindow ](https://component.tailwindow.com/)| Components created using Tailwind CSS utilities class, so it can be customized to make themes easily. |
```
Comment (Alaev) [inline readme.md:722]: Looks like the page is live

Comment (robvanbakel) [reply readme.md:722]: All I'm getting is this placeholder page with (spam) links. Are you getting a different page?

https://component.tailwindow.com

Comment (Alaev) [reply readme.md:722]: image


Looks like it's not 100% mobile friendly.

**Comment by Alaev** [comment]
Closing

**Comment by richardoey** [comment]
I see the same page like @robvanbakel. Are you sure @Alaev  you don't see the cache page? or the link we point to is different? 

https://component.tailwindow.com/

**Pull request closed without merging**